torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -1048,6 +1048,49 @@ def batch_norm_inference(input, weight, bias, mean, var, momentum, eps):
                 output_shape = node.meta["val"].size()
                 return insert_sharded_nhwc_to_nchw(g, output_tensor, output_shape)
 
+            if node.target == torch.ops.aten.slice_scatter.default:
+                tensor, src_tensor, dim, start, end, *step = args
```

Comment:
[start & end seems optional in there](https://pytorch.org/docs/stable/torch.compiler_ir.html), and maybe appeared in kwargs, maybe should have a check

Reply:
I have revised to more check on `start` and `end`